keyboards/adkb96/Readme.md

```diff
@@ -0,0 +1,17 @@
+# ADKB96
+
+![ADKB96](http://btoshop.jp/wp-content/uploads/sites/3/2019/04/ADKB96_ALL.png)  
+
+A 16x6 ortholinear keyboard kit made and sold by Bit Trade One Ltd. [More info on Web](http://bit-trade-one.co.jp/selfmadekb/adkb96/)  
+
+Keyboard Maintainer: [Bit Trade One Ltd.](http://bit-trade-one.co.jp/)  
+Hardware Supported: ADKB96 PCB,Pro Micro  
+Hardware Availability:[PCB & case Data](https://github.com/bit-trade-one/ADKB96-hardware),[BTOS Shop](http://btoshop.jp/2019/04/11/4562469772424/)  
```

Comment:
Grammar/readability fix.

```suggestion
Hardware Availability: [PCB & case Data](https://github.com/bit-trade-one/ADKB96-hardware), [BTOS Shop](http://btoshop.jp/2019/04/11/4562469772424/)  
```

Reply:
I'm sorry,I accidentally Resolved conversation before commit! I will fix it manually.